### PR TITLE
Improved Storybooks and added tag link in navigation

### LINKED ISSUE #20: fix: improve avatar group component
- [ ] max prop's logic is not correct as max should be positive number more than or equals to 1

### PR DESCRIPTION
**Pull request by Gajendra-singh-r**
Closes #20  

- [x] Added tag link in `docs`
- [x] Improved stories of all components in the `storybook`

### CODE REVIEW

File: apps/storybook/src/tab.stories.tsx [hunk 1]
```diff
@@ -6,23 +6,30 @@ const meta: Meta<typeof Tab> = {
   args: {
     size: "md",
     orientation: "horizontal",
+    variant: "line",
   },
   argTypes: {
     size: { control: "select", options: ["sm", "md", "lg"] },
+    variant: { control: "select", options: ["enclosed", "line"] },
   },
```
Comment (LuckySharma014) [inline apps/storybook/src/tab.stories.tsx:14]: Add select for orientation

File: apps/storybook/src/tab.stories.tsx [hunk 2]
```diff
@@ -6,23 +6,30 @@ const meta: Meta<typeof Tab> = {
...
 export const Variants: Story = {
-  render: ({ size, orientation }) => (
+  render: ({ size, orientation, variant }) => (
     <>
       <div
         data-cy="tab-1"
         className="w-[640px] rounded-md border py-2 dark:border-zinc-700"
       >
-        <Tab defaultValue="tab1" size={size} orientation={orientation}>
+        <Tab
```
Comment (LuckySharma014) [inline apps/storybook/src/tab.stories.tsx:27]: - Remove 2nd tab and pass all args props to 1st tab
- Also remove line that says variant=enclosed

File: apps/storybook/src/table.stories.tsx [hunk 3]
```diff
@@ -14,17 +14,19 @@ const meta: Meta<typeof Table> = {
   title: "Components / Table",
   args: {
     size: "md",
+    variant: "simple",
```
Comment (LuckySharma014) [inline apps/storybook/src/table.stories.tsx:17]: Add isBarebone to args

File: apps/storybook/src/toggle-group.stories.tsx [hunk 4]
```diff
@@ -28,3 +28,15 @@ export const Default: Story = {
     </div>
   ),
 };
+
+export const Type: Story = {
+  render: () => (
+    <div className="flex w-80 flex-col gap-6">
+      <ToggleGroup type="multiple">
```
Comment (LuckySharma014) [inline apps/storybook/src/toggle-group.stories.tsx:35]: Pass size prop from args to both functions' ToggleGroup components

**Review by LuckySharma014 — CHANGES_REQUESTED** [review]

File: apps/storybook/src/list.stories.tsx [hunk 5]
```diff
@@ -18,3 +30,37 @@ export const Default: Story = {
     </OrderedList>
   ),
 };
+
+export const Unordered_List: Story = {
```
Comment (LuckySharma014) [inline apps/storybook/src/list.stories.tsx:34]: Remove _(underscore) from function name

File: apps/storybook/src/list.stories.tsx [hunk 6]
```diff
@@ -1,14 +1,26 @@
...
+    <List>
+      <ListItem>Lorem ipsum dolor sit amet.</ListItem>
+      <ListItem>Consectetur adipiscing elit.</ListItem>
+      <ListItem>Integer molestie lorem at massa.</ListItem>
+      <ListItem>Facilisis in pretium nisl aliquet.</ListItem>
+    </List>
+  ),
+};
+
+export const Ordered_List: Story = {
```
Comment (LuckySharma014) [inline apps/storybook/src/list.stories.tsx:23]: Remove _(underscore) from function name

**Review by LuckySharma014 — CHANGES_REQUESTED** [review]

**Review by LuckySharma014 — CHANGES_REQUESTED** [review]

File: apps/storybook/src/alertdialog.stories.tsx [hunk 7]
```diff
@@ -22,18 +22,18 @@ const meta: Meta<typeof AlertDialog> = {
...
   },
 };
 export default meta;
+
 type Story = StoryObj<typeof AlertDialog>;
 
 export const Default: Story = {
   render: ({ size, isBarebone }) => (
     <AlertDialog size={size} isBarebone={isBarebone}>
       <AlertDialogTrigger>Open</AlertDialogTrigger>
```
[the remark this answers is not in the document]
Comment (Gajendra-singh-r) [reply apps/storybook/src/alertdialog.stories.tsx:34]: xl size  can not be passed in trigger

**Review by LuckySharma014 — APPROVED** [review]

**Pull request merged**